### DISCUSSION TITLE
Internationalization (i18n)

### DISCUSSION THREAD

**Discussion by sho5akimoto** [langflow-ai/langflow#1799] [Ideas]
Hi all, I want to share my idea which will be able to have much LangFlow lovers.
Now LangFlow only support English strings for User Interface like 'My Collection', 'Edit Code', "Edit your Python code. This code snippet accepts module import and a single function definition. Make sure that your function returns a string.".
'Internationalization (i18n)' is the key feature for non-coding solutions because many non-technical people who live in countries where English is not the first language. And I found a competitor start to provide 'Internationalization (i18n)' feature recently.
Note that I will be able to contribute Japanese once LangFlow provide 'Internationalization (i18n)' feature.

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
